docs/TROUBLESHOOTING.md

```diff
@@ -102,6 +102,16 @@ ControllerMode = dual
 JustWorksRepairing = confirm
 ```
 
+#### Xbox Wireless controller
+
+The newest wireless controllers from Microsoft (Xbox One and Xbox Series X/S) are known to cause a reconnect loop and not pairing with Bluez. There are some specific workarounds :
+
+- Plug your controller to a Windows 10 computer via a USB cord. Download the [xbox accessories application](https://xbox.com/accessories-app). When launching the app, it should ask you to update the firmware of the controller. When it's done, your controller should work just fine with any Linux system.
```

Comment:
It should be "Xbox Accessories application" because it's the proper name for an established application: You are explicitly referencing this application from the store.